plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationManagerImpl.kt

```diff
@@ -663,16 +664,29 @@ abstract class AnnotationManagerImpl<G : Geometry, T : Annotation<G>, S : Annota
       return false
     }
 
+    private val onMapIdleListener: OnMapIdleListener = object : OnMapIdleListener {
+      override fun onMapIdle() {
+        mapListenerDelegate.removeOnMapIdleListener(this)
+        // Remove dragging annotation from drag layer
+        Handler(Looper.getMainLooper()).postDelayed(
+          {
+            // Delay 1 second to avoid blink issue.
+            delegateProvider.getStyle { style ->
+              updateDragSource(style)
+            }
+          },
+          1000
```

Comment:
move to const

Reply:
and why 1000? isn't it too much?

Reply:
Users can't notice this delay, we need this buffer to cover the update delay, if too short, the blink may come back when there are too many annotations.

Reply:
Well, short is a pretty common word 😄 We don't know what device / how many annotations user is adding. If it's the only way to go do we need to make that delay configurable by adding getter / setter from the user side?

Reply:
IMO, we should expose this config to users unless they have a strong request on it. Would love to get enough feedback before that step.